src/sparseml/pytorch/optim/modifier_quantization.py

```diff
@@ -141,13 +158,15 @@ def __init__(
         self._quantize_embeddings = quantize_embeddings
         self._reduce_range = reduce_range
         self._quantize_linear_activations = quantize_linear_activations
-        self._exclude_module_types = exclude_module_types
 
         self._modules_to_quantize = None
         self._qat_enabled = False
         self._quantization_observer_disabled = False
         self._bn_stats_frozen = False
-
+        self._calibration_dataloader = None
+        self._calibration_function = None
+        self._num_calibration_steps = num_calibration_steps
+        self._exclude_module_types = exclude_module_types
```

Comment:
nit: move variables set from init params to group with line 160

Reply:
Addressed!